### PR TITLE
Add ConfigMaps for portal backend

### PR DESCRIPTION
**Pull request by trev-dev**
Jira: OFMCC-1860

### CODE REVIEW

File: tools/config/update-configmap.sh [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+echo Formatting public key from SOAM
+formattedPublicKey=$(cat << PUBKEY
+-----BEGIN PUBLIC KEY-----
+$(echo "$soamOneLineKey" | sed -e "s/.\{64\}/&\n/g")
+-----END PUBLIC KEY-----
+PUBKEY
+)
+echo "$formattedPublicKey"
+
+getSecret(){
```
Comment (weskubo-cgi) [inline tools/config/update-configmap.sh:45]: Is this used?

File: tools/config/update-configmap.sh [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+echo Removing key files
+rm tempPenBackendkey
+rm tempPenBackendkey.pub
+
+echo Creating config map "$APP_NAME-backend-config-map"
+oc create -n "$OPENSHIFT_NAMESPACE-$envValue" configmap \
+  "$APP_NAME-backend-$envValue-config-map" \
+  --from-literal="CLAMAV_HOST=clamav.$COMMON_NAMESPACE-$envValue.svc.cluster.local" \
+  --from-literal=CLAMAV_PORT=3310 \
```
Comment (weskubo-cgi) [inline tools/config/update-configmap.sh:62]: We should be consistent about our usage of quotes. See others below as well.

**Review by weskubo-cgi — CHANGES_REQUESTED** [review]

File: tools/config/update-configmap.sh (hunk 2 above)
Comment (trev-dev) [reply tools/config/update-configmap.sh:62]: > We should be consistent about our usage of quotes. See others below as well.

The quotation actually came from my tooling as a good suggestion, as quoting prevents accidental glob/shell expansions

File: tools/config/update-configmap.sh [hunk 3]
```diff
@@ -0,0 +1,108 @@
+set -euo pipefail
+
+envValue=$1
```
[the remark this answers is not in the document]
Comment (trev-dev) [reply tools/config/update-configmap.sh:3]: @weskubo-cgi copy-pasta.  I agree, let's upcase this

File: tools/config/update-configmap.sh (hunk 1 above)
Comment (trev-dev) [reply tools/config/update-configmap.sh:45]: Nope

File: tools/config/update-configmap.sh (hunk 2 above)
Comment (weskubo-cgi) [reply tools/config/update-configmap.sh:58]: I got it. Didn't notice that the unquoted entries have no vars. Thanks!

**Review by weskubo-cgi — APPROVED** [review]

**Review by weskubo-cgi — APPROVED** [review]

**Comment by weskubo-cgi** [comment]
Latest updates look good. Dev and Test (QA) environments have been validated.

**Pull request merged**